include/exadg/incompressible_navier_stokes/postprocessor/line_plot_data.h

```diff
@@ -242,55 +226,42 @@ struct LinePlotData
   std::vector<std::shared_ptr<Line<dim>>> lines;
 };
 
+
 template<int dim>
-struct LinePlotDataInstantaneous
+struct LinePlotData : public LinePlotDataBase<dim>
 {
-  LinePlotDataInstantaneous() : calculate(false)
-
-  {
-  }
+  TimeControlData time_control_data;
 
   void
   print(dealii::ConditionalOStream & pcout)
   {
-    line_data.print(pcout);
+    if(time_control_data.is_active)
+    {
+      this->print_base();
+      // only makes sense in unsteady case
+      time_control_data.print(pcout, true);
+    }
   }
-
-  LinePlotData<dim> line_data;
-
-  /*
-   *  activates line plot calculation
-   */
-  bool calculate;
 };
 
 template<int dim>
-struct LinePlotDataStatistics
+struct LinePlotDataStatistics : public LinePlotDataBase<dim>
 {
-  LinePlotDataStatistics()
-  {
-  }
+  TimeControlDataStatistics time_control_data_statistics;
 
   void
   print(dealii::ConditionalOStream & pcout)
   {
-    line_data.print(pcout);
-
-    if(statistics_data.calculate)
+    if(time_control_data_statistics.time_control_data.is_active)
     {
-      statistics_data.print(pcout);
+      this->print_base();
+      // only makes sense in unsteady case
+      time_control_data_statistics.time_control_data.print(pcout, true);
```

Comment:
This implies that `write_preliminary_results_...` will not be printed. Ideally, we would also introduce a print function for `time_control_data_statistics`.

Reply:
The print function is there. I just didn't call it correctly. 
[1be0d8e](https://github.com/exadg/exadg/pull/262/commits/1be0d8ee3d133733e1e6a3c80f10cb3142708837)